database.csv

```diff
@@ -183,8 +184,13 @@ id,sse,vr,status,name
 24207,0x1403606d0,0x14036ffd0,3,RE::TESNPC::UpdateNeck
 24211,0x140360c10,0x140370510,2,enhanced_reanimation::TESNPC::Activate
 24212,0x140361640,0x140370f60,3,enhanced_reanimation::TESNPC::GetActivateText
+24221,0x140363000,0x140372920,3,WhosWho::LoadArmorRelatedFunction4
+24226,0x140363CD0,0x140373630,3,TESNPC::
 24228,0x140363f20,0x140373880,4,RE::TESNPC::UpdateHead
+24232,0x140364350,0x140373CB0,3,WhosWho::LoadArmorRelatedFunction
+24233,0x140364680,0x140373FE0,3,WhosWho::LoadArmorRelatedFunction2
 24236,0x140364ff0,0x140374950,3,FEC::processArmorAttach::PerformNPCDismember
+24237,0x140364680,0x140374D60,3,WhosWho::LoadArmorRelatedFunction3
```

Comment:
@Nightfallstorm FYI, I'm getting a warning that the SSE address is incorrect. See [canonical ](https://raw.githubusercontent.com/alandtse/skyrim_vr_address_library/main/offsets-1-5-97-0.csv)which has
```csv
24237,140365400
```
If the SSE address is right, the ID should be
```csv
24233,140364680
```

Reply:
Actually I just checked and yes it's a typo. The VR matches the first item.